schedule/schedule_commands.go

```diff
@@ -459,7 +459,7 @@ func DescribeSchedule(c *cli.Context) error {
 	if sw := s.Action.GetStartWorkflow(); sw != nil {
 		item.StartWorkflow = sw
 		item.WorkflowType = sw.WorkflowType.GetName()
-		item.Input = dataconverter.CustomDataConverter().ToStrings(sw.Input)
+		item.Input = converter.GetDefaultDataConverter().ToStrings(sw.Input)
```

Comment:
Correctness: `s` is `resp.Schedule`, and `resp` was obtained using `frontendClient`, https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/schedule/schedule_commands.go#L412
and `frontendClient` is created in a way that has the gRPC interceptor set: https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/schedule/schedule_commands.go#L35